app/controllers/api/integrations_controller.rb

```diff
@@ -13,21 +13,14 @@ class Api::IntegrationsController < Api::BaseController
   rescue_from ActiveRecord::StaleObjectError, with: :edit_stale
 
   def edit
-    begin
-      @registry_policies = Policies::PoliciesListService.call!(current_account, proxy: @proxy)
-    rescue StandardError => error
-      @error = error
-    end
-    @latest_lua = current_account.proxy_logs.first
-    @deploying =  ThreeScale::TimedValue.get(deploying_hosted_proxy_key)
-    @ever_deployed_hosted = current_account.hosted_proxy_deployed_at.present?
+    set_registry_policies
   end
 
   def settings; end
 
   def update
     if @service.using_proxy_pro? && !@proxy.apicast_configuration_driven
-      proxy_pro_update
+      return redirect_to(edit_path) if proxy_pro_update
```

Comment:
I think we can remove this line, proxy_pro was removed. Some dust to clean :)

Reply:
probably remove the whole method of `proxy_pro_update`